tools/mtouch/Assembly.cs

```diff
@@ -72,14 +72,22 @@ HashSet<string> ComputeDependencies (List<Exception> warnings)
 		}
 
 		// returns false if the assembly was not copied (because it was already up-to-date).
-		public bool CopyAssembly (string source, string target, bool copy_mdb = true)
+		public bool CopyAssembly (string source, string target, bool copy_mdb = true, bool strip = false)
```

Comment:
The de-duplication process is a bit different now, instead of finding duplicates at the very end and replacing copies with symlinks, we're only copying the two versions if they're really different (this means that there won't be any symlinks, and also that since we strip at the very end, we won't be stripping any assemblies unnecessarily). This is described a bit here: 03614fc77523f3b08f5d20aeacd1e6cc1ade78fa